app/views/layouts/application.html.erb

```diff
@@ -8,6 +8,11 @@
 </head>
 <body>
 
+<p class="notice"><%= notice %></p>
+<p class="alert"><%= alert %></p>
```

Comment:
Actually I am not sure these are used at all. I think the `main` layout is used in all the places.